src/main/java/com/example/democamel/processor/ProcessorAveragesAndTotals.java

```diff
@@ -0,0 +1,44 @@
+package com.example.democamel.processor;
+
+import com.example.democamel.model.OrderReportCsv;
+import com.example.democamel.model.OrderTotals;
+import com.example.democamel.model.OrdersAggregatedByCountry;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class ProcessorAveragesAndTotals implements Processor {
```

Comment:
minor now, but good to know in general https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace

take a glance at the whole guide when you have time, too
although I don't encourage to, and I think most teams don't strictly use Google Java Style Guide, but still a style that is very close to it